modules/33acrossIdSystem.js

```diff
@@ -9,32 +9,42 @@ import { logMessage, logError } from '../src/utils.js';
 import { ajaxBuilder } from '../src/ajax.js';
 import { submodule } from '../src/hook.js';
 import { uspDataHandler, coppaDataHandler, gppDataHandler } from '../src/adapterManager.js';
+import { getStorageManager, STORAGE_TYPE_COOKIES, STORAGE_TYPE_LOCALSTORAGE } from '../src/storageManager.js';
```

Comment:
publishers who evaluated your module before and arent comfortable with these functions might consider this breaking in a minor version. Please put all this new functionality behind a configuration flag.

Reply:
@patmmccann When you say configuration flag, do you mean a module configuration flag that's passed to pbjs.setConfig() or a compile-time feature flag that lives in features.json?

Can we default its value to true if unspecified as long as we provide clear messaging and documentation for this fiag to existing publishers?

Reply:
No; you should flip the default in a major version.

Reply:
Configuration flag has been added, for the moment the default value is false and will be changed to true in a major version.

Reply:
excellent, that branch will open in April